Yukki/Plugins/Ping.py

```diff
@@ -6,7 +6,7 @@
 from pyrogram import Client, filters
 from pyrogram.types import Message
 
-from Yukki import BOT_USERNAME, MUSIC_BOT_NAME, app, boottime
+from Yukki import BOT_USERNAME, MUSIC_BOT_NAME, PING_IMG app, boottime
```

Comment:
You should put a , between PING_IMG and app.

Reply:
yeah missed

Reply:
yeah missed

Reply:
yeah missed